lang/java/BUILD.md

```diff
@@ -44,6 +46,12 @@ To perform "fast" build, which skips tests and all code quality enforcement tool
 
     mvn clean install -DskipTests -TC1 -P!code-quality
 
+
+  * Please follow the following wiki for all running all java builds [Java Build Wiki](https://cwiki.apache.org/confluence/display/REEF/Linux), the concept of ulimit increase allows more file descriptors to be opened for some of the unit tests in REEF
```

Comment:
I think the build instructions should be self-contained in this file. It is part of our releases and version controlled, so it should always fit the version of REEF someone is using. We actually used to have build instructions on the wiki which we moved to the repository for exactly that reason.